.github/CODEOWNERS

```diff
@@ -396,6 +396,7 @@
 # Story project.
 /core/controllers/story*.py @aks681
 /core/domain/story*.py @aks681
+/core/domain/subtopic_*.py @aks68
```

Comment:
Invalid userID, please add correct the userID @aks681 (Missing 1 at the end.)

Reply:
Updated!